### DISCUSSION TITLE
G-helper is just awesome

### DISCUSSION THREAD

**Discussion by Moorole** [seerge/g-helper#1136] [General]
Just want to write how nice this little app works:
Before I was fighting with AC hating it and struggling to set my fans properly until I gave up deleting AC completely.
I was left with a loud laptop, fans either off or shouting at me even in idle/light load. sometimes uncabled it to have silence.
Now my laptops runs nicely with fans of during light load, battery protected with charge at 65%, undervolt to -35 and gpu also saving loads of battery, refresh rate and keyboard lighting switching with plugging, and having a smooth experience during gaming where fans go up slowly down also. All in all i am so happy to now have a well manufactured machine from asus (where i cannot complain, they build nice hardware) and a software which from my point of view doesn't let me behind with any wishes. 
I think we should promote this nice peace a little on reddit and other platforms to get more reward for the creators and also draw some attention by Asus and other manufacturers to make better software. 
Thank you soo much, seerge and others you are putting there time into this project 
:)))))))

**Comment by ocke** [comment]
I want to add that I completely agree. Great piece of software @seerge!
